readthedocs/notifications/messages.py

```diff
@@ -96,6 +96,19 @@ def get_rendered_body(self):
         ),
         type=ERROR,
     ),
+    Message(
+        id=BuildAppError.UPLOAD_FAILED,
+        header=_("There was a problem while updating your documentation"),
```

Comment:
Updating or uploading? 🤔 

```suggestion
        header=_("There was a problem while uploading your documentation"),
```

Reply:
Yeah, uploading is an internal term to us. The user doesn't need to know we upload to S3 -- the immediate end result that the user notices is that their documentation wasn't _updated_.